climate_scanner/entity_network/entity_extraction.py

```diff
@@ -15,8 +15,10 @@
 import requests
 import os
 
+
+
 # Set via environment variables
-user_key = os.getenv('user_key')
+user_key = 'enter user key here'
```

Comment:
I want us to actually start using a package called python dotenv. Check out how this works, essentially you have a hidden file in the directory (.env), which sets the environment variables (authorisations) you need to run your program. It's easy to edit and maintain, and we configure the gitignore file to prevent the actual authorasations from ever accidentally being uploaded to github.